src/login/login.resource.tsx

```diff
@@ -1,20 +1,10 @@
+import { openmrsFetch } from "@openmrs/esm-api";
+
 export function performLogin(username, password) {
   const token = window.btoa(`${username}:${password}`);
-  return window
-    .fetch(`/openmrs/ws/rest/v1/session`, {
-      method: "GET",
-      headers: {
-        Accept: "application/json",
```

Comment:
Notice how `/Accept` is no longer needed -- done automatically in `@openmrs/esm-api`